webapi/src/main/twirl/queries/sparql/v2/getResourcePropertiesAndValuesGraphDB.scala.txt

```diff
@@ -71,7 +76,8 @@ CONSTRUCT {
     ?resource knora-base:hasValue ?valueObject ;
         ?resourceValueProperty ?valueObject .
 
-    ?valueObject ?valueObjectProperty ?valueObjectValue .
+    ?valueObject ?valueObjectProperty ?valueObjectValue ;
+        knora-base:valueHasUUID ?currentValueUUID .
```

Comment:
Do I understand this correctly: if given a value UUID, only return the value matching the given value UUID.

1. if a timestamp is provided, use the existing logic to determine which previous version of the current value (that has the UUID) has to be returned
2. if no timestamp is provided, just serve the current version of the matching value

Case 1 uses the same logic used when doing a full resource request with a timestamp, right? The difference is that it only serves one value.

Reply:
Yes, exactly.

Reply:
great, thanks